CHANGES.md

```diff
@@ -25,6 +25,8 @@
 
 - The function `to_file` now adds a newline at the end of the generated file. An
   optional argument allows to return to the original behaviour (#124, @panglesd)
+- The stream_from_* and stream_to_* functions now use a Seq.t instead of a Stream.t,
```

Comment:
```suggestion
- The `stream_from_*` and `stream_to_*` functions now use a Seq.t instead of a Stream.t,
```

Reply:
Done, also quoted the other identifiers.